egs2/swahili/asr1/local/valid-iwslt.txt

```diff
@@ -0,0 +1,869 @@
+iwslt_1-828fade7fed32c93944ace4965f81a48__1573474880.3137 Julia anasema Tommy atakula chakula cha mchana na yeye.
```

Comment:
Can we load this somewhere else instead of host it in the repo?

Reply:
Moved to zenodo.